engine/tests/provider/provider_test.cpp

```diff
@@ -37,16 +37,39 @@ void create_dummy_file(std::string content, std::string filename){
 	outfile.close();
 }
 
+void create_folder_test()
```

Comment:
Just because the api returns true or false doe snot mean the directory was properly created. You need to make sure it actually exists using file apis

Reply:
Done